src/utils/axiosWithAuth.js

```diff
@@ -1,12 +1,12 @@
 import axios from "axios";
-
 export const axiosWithAuth = () => {
   const token = localStorage.getItem("token");
 
   return axios.create({
+    baseURL: `process.env.BASEURL`,
```

Comment:
I've seen a baseURL in axios but I don't recall seeing it with the env property before. Pretty cool.

Reply:
Right!? I thought this was useful to set it dynamically based on the environment. 
--Locally, we can set it to be the local host and whatever port the backend is set to spin up on. ---In production, we can set it to the Hosted backend url on Elastic Beanstalk.
🌠  '